components/molecules/RecommendedRepoCard/recommended-repo-card.tsx

```diff
@@ -52,7 +52,7 @@ const RecommendedRepoCard = ({ fullName, className }: RecommendedRepoCardProps):
           <div className="flex flex-col gap-2 pb-3">
             <a
               className="text-xl font-semibold"
-              href={`https://app.opensauced.pizza/hot/repositories/filter/${fullName}`}
+              href={`https://app.opensauced.pizza/hot/dashboard/filter/${fullName}`}
```

Comment:
> Thanks for the fix @sourabpramanik but this should be caught by a redirect, right @OgDev-01?

@brandonroberts, thanks for catching that. this will be caught in the redirect, but will hit a 404 since the url pattern doesn’t match any dynamic route.

@sourabpramanik, here is a quick refactor to fix this
```suggestion
              href={`https://app.opensauced.pizza/${name}/dashboard/filter/recent`}
```
The goal is to enable users get clear insights on the repo they click on from the recommendations